hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/backend/provider/HttpBackendProvider.java

```diff
@@ -21,6 +21,11 @@
 import org.hypertrace.traceenricher.enrichment.enrichers.BackendType;
 
 public class HttpBackendProvider implements BackendProvider {
+  private static final String COLON = ":";
+  private static final int DEFAULT_HTTP_PORT = 80;
```

Comment:
is 8080/8000 also need in the list?

Reply:
These are not default ports. User need to explicitly provide these port as part of their requests. Only handling case when in some requests ports(default ports) are explicitly provided while in some case no ports(default ports implicitly used) are specified.